src/pseudopeople/configuration/validator.py

```diff
@@ -13,7 +13,7 @@
 from pseudopeople.schema_entities import DatasetSchema
 
 
-def validate_overrides(overrides: Dict, default_config: LayeredConfigTree) -> None:
+def validate_overrides(overrides: Any, default_config: LayeredConfigTree) -> dict:
```

Comment:
Do we still want to return a dict here? You're not using the return where you call this function.

Reply:
fixed